acount-rest-service/domain/model/budgets.go

```diff
@@ -27,3 +27,23 @@ func NewStandardBudgets(standardBudgetByCategoryList []StandardBudgetByCategory)
 func NewCustomBudgets(customBudgetByCategoryList []CustomBudgetByCategory) CustomBudgets {
 	return CustomBudgets{CustomBudgets: customBudgetByCategoryList}
 }
+
+func (b StandardBudgets) ShowBudgetsList() []int {
+	var budgetsList []int
+
+	for i := 0; i < len(b.StandardBudgets); i++ {
+		budgetsList = append(budgetsList, b.StandardBudgets[i].Budget)
+	}
```

Comment:
```suggestion
	budgetsList := make([]int, len(b.StandardBudgets))

	for i := 0; i < len(b.StandardBudgets); i++ {
		budgetsList[i] = b.StandardBudgets[i].Budget
	}
```

長さが決まっている slice を作成する際は make で長さを指定して作成するとメモリ確保が 1 回で済む様になるのでいいと思います！

Reply:
アドバイスありがとうございます！
すごく参考になります！

Reply:
ed45470
修正しました。